GLOSSARY.md

```diff
@@ -161,10 +163,42 @@ let onboardingMessage = helloWorld; // Type: string
 When the `helloWorld` constant was re-used in a mutable variable `onboardingMessage` the type which was set is an
 expanded version of `"Hello World"` which went from one value ever, to any known string.
 
-- `Transient` - unsure
+- `Transient` - a symbol created in the checker.
+
+  The checker creates its own symbols for unusual declarations:
+
+  1. Cross-file declarations
+
+  ```ts
+  // @filename: one.ts
+  interface I { a }
+  // @filename: two.ts
+  interface I { b }
+  ```
+
+  The binder creates two symbols for I, one in each file. Then the checker creates a merged symbol that has both declarations.
+
+  2. Synthetic properties
+
+  ```ts
+  type Nats = Record<'one' | 'two', number>
+  ```
+
+  The binder doesn't create any symbols for `one` or `two`, because those properties don't exist until the Record mapped type creates them.
+
+  3. Complex JS patterns
+
+  ```js
+  var f = function g() {
+    g.expando1 = {}
+
+  }
+  f.expando2 = {}
+  ```
 
+  People can put expando properties on a function expression inside or outside the function, using different names.
 - `Partial Type` -
-- `Synthetic` -
+- `Synthetic` - a property that doesn't have a declaration in source.
```

Comment:
Is `Synthetic` here only referring to a property? I found the use of 'property' here sort of vague... is it referring to properties like `syntheticOrigin`?

Reply:
I don't disagree with the feedback, but I'm making it non-blocking so I can merge